charts/coatrack/old_readme.md

```diff
@@ -0,0 +1,109 @@
+# Generic Microservice Chart #
```

Comment:
Whats the purpose of this "old_readme"? If you want to extend the generated one, you can provide the added content via templating, check f.e. https://github.com/FIWARE/helm-charts/blob/main/charts/endpoint-auth-service/README.md.gotmpl